Tools/Taxonomy/Common/Constants.cs

```diff
@@ -46,20 +52,74 @@ private static class Guid
             ReleaseDate = "2021-07-20",
         };
 
-        public static TaxonomyData CWE_Comprehensive_V43 = new TaxonomyData
```

Comment:
>CWE_Comprehensive_V43 [](http://example.com/codeflow?start=35&length=21)

I changed this from individual var to a dictionary so i can use it easier in file `CweTaxonomyGenerator.cs`